packages/vscode-extension/src/commandHandlers/params.ts

```diff
@@ -55,6 +60,27 @@ export async function addParameter(): Promise<void> {
   await evaluateParam(paramName, paramValue);
 }
 
+export async function modifyParameter(paramItem: ParameterItem): Promise<void> {
+  const connected = await isConnected();
+  if (!connected) {
+    await window.showErrorMessage(
+      'You need to be connected to neo4j to edit parameters.',
+    );
+    return;
+  }
+  const existingParam = getParameter(paramItem.id);
+  if (!existingParam) {
+    return;
+  }
+  const paramValue = await window.showInputBox({
+    prompt: 'Parameter value',
+    value: existingParam.evaluatedStatement,
+    ignoreFocusOut: true,
+  });
+
+  await evaluateParam(paramItem.id, paramValue);
```

Comment:
Similar to the add parameter functionality, we should add the check:

```ts
  if (!paramValue) {
    await window.showErrorMessage('Parameter value cannot be empty.');
    return;
  }
```

before the `evaluateParam` call.

Reply:
I also realized the `if (!paramName)` and `if(!paramValue)` checks in the `addParameter()` method are missing the early return, so fix that in the PR 🙏.

Probably that's why we were saying `wrong format for the parameter` messages pop up when it didn't make sense.